documentation/src/main/asciidoc/cheat-sheets/cache-configuration.asciidoc

```diff
@@ -0,0 +1,435 @@
+
+[source,xml]
+----
+<distributed-cache mode="SYNC">
+  <encoding>
+    <key media-type="application/x-protostream"/>
+    <value media-type="application/x-protostream"/>
+  </encoding>
+</distributed-cache>
+----
+
+.Expiration
+Controls how long {brandname} keeps entries in a cache and takes effect across clusters.
+
+Lifespan:: The maximum amount of time, in milliseconds, that cache entries can exist.
+If you set the value to -1, entries never expire.
+Lifespan uses a `creation` timestamp and the value for the `lifespan` property to determine when entries expire.
+The creation time for cache entries provides a value that is consistent across clustered caches.
+
+Max-idle:: The maximum amount of time, in milliseconds, that cache entries can remain idle.
+If no operations are performed on entries within the maximum idle time, the entries expire across the cluster.
+If you set the value to -1, entries never expire.
+
+Maximum idle uses a `last used` timestamp and the value for the `max-idle` property to determine when entries expire.
+The last access time for entries is not always the same on all nodes.
+To ensure that entries have the same relative access times across clusters, {brandname} sends touch commands to all owners when keys are accessed.
+This synchronous behavior increases latency of client requests.
+
+TIP: The {brandname} team recommends setting the value for `lifespan` to be at least double of the value that you set for the `max-idle` property. Configuring `max-idle` close to `lifespan` has performance drawbacks.
+
+[source,xml]
+----
+<distributed-cache>
+  <expiration lifespan="5000"
+              max-idle="1000" />
+</distributed-cache>
+----
+
+.Statistics
+Start collecting statistics for your cache.
+Keep disabled for optimal performance.
+
+[source,xml]
+----
+<distributed-cache statistics="true" />
+----
+
+== {brandname} capabilities
+* Bounded
+* Indexed
+* Authorization
+* Backups
+* Transactional
+* Persistent
+
+== Bounded caches
+Eviction controls the amount of data that {brandname} keeps in memory and takes effect for each node.
+Because not all nodes evict the same entries you should use eviction with persistent storage to avoid data inconsistency.
+
+.Eviction
+Control the size of the cache in one of the following ways:
+
+Max size:: Maximum size of the data container in bytes.
+Eviction occurs when the approximate memory usage of the data container exceeds the maximum size.
++
+[source,xml]
+----
+<distributed-cache>
+  <memory max-size="12MB" when-full="REMOVE"/>
+</distributed-cache>
+----
+
+Max count:: Maximum number of entries.
+Eviction occurs after the container size exceeds the maximum count.
++
+[source,xml]
+----
+<distributed-cache>
+	<memory max-count="124" when-full="REMOVE"/>
+</distributed-cache>
+----
+
+.Eviction strategies
+* `REMOVE` {brandname} performs eviction. This is the default strategy when you define a size for the data container.
```

Comment:
Note that REMOVE will not have an effect to the persistence, the entry is removed from the memory and if requested again it will be reloaded from the persistence